src/modules/weather.js

```diff
@@ -0,0 +1,17 @@
+export default class Weather {
+  constructor(city) {
+    this.apiKey = '88dd50190cb7c4a2c8e55d61d1341d41';
+    this.city = city;
+  }
+
+  getWeather = async () => {
+    const apiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${this.city}&units=metric&appid=${this.apiKey}`;
+    const response = await fetch(apiUrl, { mode: 'cors' });
```

Comment:
* [ ] It has error handling for `promises` or `async/await` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/error_handling_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))
* Let us use `try and catch` block, based on this we can display error on the UI when the data fetching fails as mention above.